packages/optimize/js/index.js

```diff
@@ -0,0 +1,32 @@
+/*
+Copyright 2021 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+
+@flow
+@format
+*/
```

Comment:
We need to add them as a comment in the files so that the flow tool will consider this file. In absence of commented flow, the file will be ignored by flow. The pattern followed in all the js files(including other RN packages) is to add them in the same commented block as license header.